tox.ini

```diff
@@ -1,7 +1,7 @@
 [flake8]
 count = True
 #default_ignorelist: E121,E123,E126,E226,E24,E704,W503,W504
-ignore = U101
+ignore = U101, E251
```

Comment:
Any specific reason to ignore E251 as well?


```
Unexpected spaces around keyword / parameter equals (E251)

There should be no spaces before or after the = in a function definition.
```

Reply:
Just that I got it confused with something else...